syntax = "proto3";

message Inference {
    float confidence = 1;
    string class_name = 2;
    int32 class_id = 3;
    repeated float bounding_box = 4;
}

message InferenceList {
    string camera_name = 1;
    repeated Inference inferences = 2;
}